plugins/bufbuild/knit-ts/v0.0.1/buf.plugin.yaml

```diff
@@ -0,0 +1,13 @@
+version: v1
+name: buf.build/bufbuild/knit-ts
+plugin_version: v0.0.1
+source_url: https://github.com/bufbuild/knit-ts
+description: Generates Knit schemas needed to use the Knit client in web browsers and Node.js.
+output_languages:
+  - typescript
+registry:
+  npm:
```

Comment:
Does it have any runtime `deps`?

Reply:
None

Reply:
The demo depends on `@bufbuild/knit` - is that not needed?

https://github.com/bufbuild/knit-demo/blob/8766e187d5730da22783a22c65a398cbd90d7f21/ts/package.json#L13-L15

Reply:
The generated code doesn't import it, this should be only if the generated code uses it right?

Reply:
Yep.